conf/modules/pano_unwrap.xml

```diff
@@ -0,0 +1,82 @@
+    <define name="PANO_UNWRAP_FORWARD_DIRECTION" value="270.0" description="Forward direction in raw image measured counterclockwise from right [deg]"/>
+    <define name="PANO_UNWRAP_FLIP_HORIZONTAL" value="TRUE|FALSE (default)" description="Flip output image horizontally"/>
+    <define name="PANO_UNWRAP_VERTICAL_RESOLUTION" value="0.18" description="Vertical resolution of region of interest [fraction of raw image height/rad]"/>
+    <define name="PANO_UNWRAP_DEROTATE_ATTITUDE" value="TRUE|FALSE (default)" description="Derotate the image, i.e. keep the panorama aligned with the horizon"/>
+    <define name="PANO_UNWRAP_WIDTH" value="640" description="Width of the unwrapped image [px]"/>
+    <define name="PANO_UNWRAP_HEIGHT" value="0" description="Height of the unwrapped image [px]. Set to 0 to determine automatically."/>
+    <define name="PANO_UNWRAP_OVERWRITE_VIDEO_THREAD" value="TRUE (default)|FALSE" description="Write unwrapped image to the video thread"/>
+    <define name="PANO_UNWRAP_FPS" value="0" description="Maximum FPS (0: unlimited)"/>
+  </doc>
+  <settings>
+    <dl_settings>
+      <dl_settings name="pano_unwrap">
+        <dl_setting shortname="center_x" var="pano_unwrap.center.x" min="0.0" step="0.01" max="1.0"/>
+        <dl_setting shortname="center_y" var="pano_unwrap.center.y" min="0.0" step="0.01" max="1.0"/>
+        <dl_setting shortname="radius_bottom" var="pano_unwrap.radius_bottom" min="0.0" step="0.01" max="1.0"/>
+        <dl_setting shortname="radius_top" var="pano_unwrap.radius_top" min="0.0" step="0.01" max="1.0"/>
+        <dl_setting shortname="forward_direction" var="pano_unwrap.forward_direction" min="0" step="1" max="360"/>
+        <dl_setting shortname="flip_hor" var="pano_unwrap.flip_horizontal" type="uint8" values="FALSE|TRUE" min="0" step="1" max="1"/>
+        <dl_setting shortname="v_resolution" var="pano_unwrap.vertical_resolution" min="-0.2" step="0.01" max="0.2"/>
+        <dl_setting shortname="derotate" var="pano_unwrap.derotate_attitude" type="uint8" values="FALSE|TRUE" min="0" step="1" max="1"/>
+        <dl_setting shortname="width" var="pano_unwrap.width" min="1" step="1" max="1920"/>
+        <dl_setting shortname="height" var="pano_unwrap.height" min="0" step="1" max="1080"/>
+        <dl_setting shortname="overwrite_video" var="pano_unwrap.overwrite_video_thread" type="uint8" values="FALSE|TRUE" min="0" step="1" max="1"/>
+        <dl_setting shortname="show_calibration" var="pano_unwrap.show_calibration" type="uint8" values="FALSE|TRUE" min="0" step="1" max="1"/>
+      </dl_settings>
+    </dl_settings>
+  </settings>
+  <depends>video_thread</depends>
+  <header>
+    <file name="pano_unwrap.h"/>
+  </header>
+  <init fun="pano_unwrap_init()"/>
+  <makefile>
+    <file name="pano_unwrap.c"/>
+    <define name="CV_ALLOW_VIDEO_TO_CHANGE_SIZE" value="1"/>
```

Comment:
When you define this, then all modules that do not support changing size will still crash if I am not mistaken...